src/FakeItEasy/StrictFakeOptions.cs

```diff
@@ -0,0 +1,37 @@
+namespace FakeItEasy
+{
+    using System;
+
+    /// <summary>
+    /// Options for strict fakes.
+    /// </summary>
+    [Flags]
+    public enum StrictFakeOptions
+    {
+        /// <summary>
+        /// Default behavior. No unconfigured method can be called, including those
+        /// inherited from <see cref="Object" />.
+        /// </summary>
+        None = 0,
+
+        /// <summary>
+        /// Calls to <see cref="Object.Equals(object)"/> are allowed.
+        /// </summary>

```

Comment:
Should we mention that they'd behave just as if the fake weren't strict?

Reply:
Sure!